consul-secure-gossip-vault/step1.md

```diff
@@ -0,0 +1,64 @@
+For this lab, you will use the Vault KV secrets engine.
+
+First, enable a new secret engine called `kv` at path `kv-v1`
+
+`vault secrets enable -path="kv-v1" kv`{{execute T1}}
+
+Example output:
+
+```
+Success! Enabled the kv secrets engine at: kv-v1/
+```
+
+Once the secret engine is enabled, verify it this using
+the following command:
+
+`vault secrets list -detailed`{{execute T1}}
```

Comment:
This command is really great for troubleshooting, but in the little katacoda terminal the output is a little difficult to read. Can we just list the engines without all the details?